File: benchmarks/eq/loop_rec/loop_rec_orig.c
int f(int m) {
    int result;

    if (m > 0) {
        result = tr(m - 1);
        result = result + m;
    } else {
        result = 0;
    }

    return result;
}

int tr(int n) {
    int result;
    int i;

    i = 0;
    result = 0;
    while ((i < n)){
        result = result + i;
    }

    return result;
}

File: benchmarks/eq/loop_rec/loop_rec_orig.py
def loop_rec(m):
    if (m > 0):
        result = tr(m-1)
        result = result + m
    else:
        result = 0;

    return result

def tr(n):

    i = 0
    result = 0

    while (i < n):
        result = result + i

    return result
